### PR TITLE
New package: gtkcord3-0.0.4

### PR DESCRIPTION
**Pull request by phossil**
See the [Package Request](https://github.com/void-linux/void-packages/issues/22568) for details.

### CODE REVIEW

File: srcpkgs/gtkcord3/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'gtkcord3'
+pkgname=gtkcord3
+version=0.0.4
+revision=1
+build_style=go
+go_import_path=github.com/diamondburned/gtkcord3/
+makedepends="libhandy-devel gtk+3-devel pkg-config"
+depends="libhandy gtk+3"
```
Comment (abenson) [inline srcpkgs/gtkcord3/template:8]: Don't need this here

File: srcpkgs/gtkcord3/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'gtkcord3'
+pkgname=gtkcord3
+version=0.0.4
+revision=1
+build_style=go
+go_import_path=github.com/diamondburned/gtkcord3/
+makedepends="libhandy-devel gtk+3-devel pkg-config"
```
Comment (abenson) [inline srcpkgs/gtkcord3/template:7]: pkg-config should be hostmakedepends

File: srcpkgs/gtkcord3/template (hunk 1 above)
Comment (phossil) [reply srcpkgs/gtkcord3/template:9]: I unfortunately didn't see this until after I forced pushed.

**Comment by ericonr** [comment]
```
diff --git a/srcpkgs/gtkcord3/template b/srcpkgs/gtkcord3/template
index 35b7afddb3..d4dd3714cc 100644
--- a/srcpkgs/gtkcord3/template
+++ b/srcpkgs/gtkcord3/template
@@ -6,10 +6,10 @@ build_style=go
 go_import_path=github.com/diamondburned/gtkcord3/
 hostmakedepends="pkg-config"
 makedepends="libhandy-devel gtk+3-devel"
-depends="libhandy gtk+3"
 short_desc="Gtk3 Discord client in Golang"
 maintainer="Phosu Parsons <founderofjp@gmail.com>"
 license="GPL-3.0-only"
 homepage="https://github.com/diamondburned/gtkcord3"
 distfiles="https://github.com/diamondburned/gtkcord3/archive/v${version}.tar.gz"
 checksum=704c623393d79cc8abb68ac59a5b243576df6857626276ec3fdf15fc9b7c753f
+LDFLAGS="-fuse-ld=bfd"
```

This solves the cross issue and removes the unnecessary `depends`.

**Comment by phossil** [comment]
Thanks, that seems to have worked.

**Comment by phossil** [comment]
Closed due to inactivity.
Package template in case anyone else wants to try submitting it:
```
# Template file for 'gtkcord3'
pkgname=gtkcord3
version=0.0.4
revision=1
build_style=go
go_import_path=github.com/diamondburned/gtkcord3/
hostmakedepends="pkg-config"
makedepends="libhandy-devel gtk+3-devel"
short_desc="Gtk3 Discord client in Golang"
maintainer="Joe Example <joe@example.com>"
license="GPL-3.0-only"
homepage="https://github.com/diamondburned/gtkcord3"
distfiles="${homepage}/archive/v${version}.tar.gz"
checksum=704c623393d79cc8abb68ac59a5b243576df6857626276ec3fdf15fc9b7c753f
LDFLAGS="-fuse-ld=bfd"
```

**Pull request closed without merging**